From 960ee687cf4b511f7c466380c9c0dbb14623d512 Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Fri, 30 Sep 2016 17:24:14 -0700
Subject: [PATCH 1902/2748] YARN-5693. Reduce loglevel to Debug in
 ContainerManagementProtocolProxy and
 AMRMClientImpl (yufeigu via rkanter)

(cherry picked from commit 2549ee9d4c4ddd3ebccdebb8623df30c0a8f27d2)
(cherry picked from commit 56ec975ea97d0de387e9bd011d57e69f9e210099)

Change-Id: I438db7c90cd4e7cd52ece50c7afcfa94aa7e372e
---
 .../yarn/client/api/impl/AMRMClientImpl.java       |   10 ++++++----
 .../api/impl/ContainerManagementProtocolProxy.java |    7 +++++--
 2 files changed, 11 insertions(+), 6 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/AMRMClientImpl.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/AMRMClientImpl.java
index ab8aaa8..a1f8c0f 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/AMRMClientImpl.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/AMRMClientImpl.java
@@ -354,10 +354,12 @@ protected void removePendingReleaseRequests(
   protected void populateNMTokens(List<NMToken> nmTokens) {
     for (NMToken token : nmTokens) {
       String nodeId = token.getNodeId().toString();
-      if (getNMTokenCache().containsToken(nodeId)) {
-        LOG.info("Replacing token for : " + nodeId);
-      } else {
-        LOG.info("Received new token for : " + nodeId);
+      if (LOG.isDebugEnabled()) {
+        if (getNMTokenCache().containsToken(nodeId)) {
+          LOG.debug("Replacing token for : " + nodeId);
+        } else {
+          LOG.debug("Received new token for : " + nodeId);
+        }
       }
       getNMTokenCache().setToken(nodeId, token.getToken());
     }
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
index a7a4a2b..db8e4f5 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/ContainerManagementProtocolProxy.java
@@ -78,8 +78,11 @@ public ContainerManagementProtocolProxy(Configuration conf,
           YarnConfiguration.NM_CLIENT_MAX_NM_PROXIES
               + " (" + maxConnectedNMs + ") can not be less than 0.");
     }
-    LOG.info(YarnConfiguration.NM_CLIENT_MAX_NM_PROXIES + " : "
-        + maxConnectedNMs);
+
+    if (LOG.isDebugEnabled()) {
+      LOG.debug(YarnConfiguration.NM_CLIENT_MAX_NM_PROXIES + " : " +
+          maxConnectedNMs);
+    }
 
     if (maxConnectedNMs > 0) {
       cmProxy =
-- 
1.7.9.5

